### ISSUE TITLE
New Project

### ISSUE THREAD

**Issue by Sandertv** [BlockHorizons/StackerPE#1]
@TheDiamondYT1 or @xBeastMode, is one of you willing to make a start on this plugin? If you don't know what it is, it's basically a plugin where players can sit on top of eachother and can punch eachother off, which gives them a boost. There's probably videos of it that explain it better than I can.

Is anybody of you willing to make a start on this project?

**Comment by TheDiamondYT1** [comment]
I tried this a while ago, but got confused with knocking back the player. I'll work on some of it now.

**Comment by TheDiamondYT1** [comment]
I got the base of it done, will push soon. If were going to continue making entity related plugins we should probably contribute to the entity rewrite in PocketMine, which makes our life easier too in the long run.

**Comment by Sandertv** [comment]
Agreed, that could make things a lot smoother. We could definitely look into that.

**Comment by TheDiamondYT1** [comment]
I've pushed it and left out the entity code for now
